src/renderer/dx/DxRenderer.cpp

```diff
@@ -251,6 +416,16 @@ DxEngine::~DxEngine()
             default:
                 THROW_HR(E_NOTIMPL);
             }
+
+            if (_retroTerminalEffects)
+            {
+                const HRESULT hr = _SetupTerminalEffects();

```

Comment:
Does this need an operating system version guard? the WPF terminal control wants to work on Windows 7.

Reply:
I don't think there's any dependency on OS version. Just directx which the terminal already depends on.